test/clojure-mode-convert-collection-test.el

```diff
@@ -28,6 +28,20 @@
 (require 'clojure-mode)
 (require 'buttercup)
 
+(defmacro when-refactoring-it (description before after &rest body)
```

Comment:
Isn't this already in the test helper? Why duplicate it here?

Reply:
See https://github.com/clojure-emacs/clojure-mode/pull/532#discussion_r301415357